packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -2,15 +2,52 @@
 
 The terra-functional-testing library is a utility for developing automation tests. The library extends the [WebdriverIO Framework](https://webdriver.io/) to facilitate automating accessibility and functional testing for Terra projects.
 
-## Viewports
+## Getting Started
 
-Tests can be executed in any of the following supported viewports:
-- 'tiny' - width: 470; height: 768
-- 'small' - width: 622; height: 768
-- 'medium' - width: 838; height: 768
-- 'large' - width: 1000; height: 768
-- 'huge' - width: 1300; height: 768
-- 'enormous' - width: 1500; height: 768
+To get started ensure [docker](https://www.docker.com/) has been installed. The terra-functional-testing library uses docker to setup and run tests in a containerized environment.
+
+Install with npm:
+
+```
+npm install --save-dev @cerner/terra-functional-testing 
+```
+
+## Test Runner
+
+A test runner is provided for local development and should be invoked using the [terra-cli](dev_tools/terra-cli/terra-cli).
+
+### Options
+
+| Option Name | Type   | Required | Default                 | Description                                                                                   |
+|-------------|--------|----------|-------------------------|-----------------------------------------------------------------------------------------------|
+| config      | string | false    |                         | A file path to the test runner configuration.                                                 |
+| formFactors | array  | false    |                         | A list of form factors for the test run. One of tiny, small, medium, large, huge, or enormous |
+| locales     | array  | false    | ['en']                  | A list of language locales for the test run.                                                  |
+| themes      | array  | false    | ['terra-default-theme'] | A list of themes for the test run.                                                            |
+
+The following example will run the test suite a total of four times. Once for each permutation of the specified locales and form factors.
+
+package.json
+```json
+{
+  "scripts": {
+    "wdio": "npm run terra -- wdio --locales en fr --formFactors tiny huge"
```

Comment:
If passed like the previous implementation, will it still run as expected?

Reply:
No, the old way would not work. yargs interprets arrays by arguments delimited by spaces:

https://github.com/yargs/yargs/blob/fd66d8e95f14302a376b7030e8d8dae869af

We discussed this early in the WDIO sync and decided to move forward with how yargs interprets arrays. We could probably do some custom parsing like we're doing today and opt out of the yargs array support, it would just depend on how much value we thought that would have. Array data structures don't really "exist" on the command line. Our current custom strategy mutates a single string argument and coerces it into an array.